README.md

```diff
@@ -26,14 +26,18 @@ in the bundle (perhaps because of out-of-date dependencies).
 
 ## Options
 
+There are three output formats available - JSON, TSV and HTML. By default (`source-map-explorer foo.min.js`), source-map-explorer writes HTML to a temporary file and opens it in your default browser. But if one of formats specified (`source-map-explorer foo.min.js --json`) source-map-explorer outputs result in specified format to stdout. If filename specified (`source-map-explorer foo.min.js --json sme/result.json`) result is saved to the file  
```

Comment:
Some suggestions on wording:

```
There are three output formats available: HTML, JSON and TSV. By default (`source-map-explorer foo.min.js`), source-map-explorer writes HTML to a temporary file and opens it in your default browser. But if one of the other formats is specified (`source-map-explorer foo.min.js --json`) source-map-explorer outputs results to stdout. If a filename is specified (`source-map-explorer foo.min.js --json sme/result.json`) the result is saved to the file.
```

Alternatively, consider replacing the paragraph with a list of the possible invocations and letting the examples speak for themselves:

```
# default behavior:
source-map-explorer foo.min.js # write HTML to a temp file and open it in your browser

# Write output in specific formats to stdout:
source-map-explorer foo.min.js --html
source-map-explorer foo.min.js --json
source-map-explorer foo.min.js --tsv

# Write output in specific formats to a file:
source-map-explorer foo.min.js --html result.html
source-map-explorer foo.min.js --json result.json
source-map-explorer foo.min.js --tsv result.tsv
```

Reply:
Replaced by  a list of the possible invocations